### PR TITLE
Add a Svelte store and a counter demo that uses it

### PR DESCRIPTION
**Pull request by ecstatic-morse**
fac20389134168323d6cb4e65c4c562037dfb1d6 fixes a `tsc` error during build. Not sure why CI didn't catch it?

The rest is just a simple [custom store](https://svelte.dev/tutorial/custom-stores) along with a Vite+Svelte default project tweaked to work with `automerge-repo`. I don't know if you ultimately want a demo for every frontend framework, but I think having the store at least is useful.

### CODE REVIEW

File: packages/automerge-repo-demo-counter-svelte/package.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    "check": "svelte-check --tsconfig ./tsconfig.json"
+  },
+  "dependencies": {
+    "svelte": "^3.49.0",
+    "@automerge/automerge": "^2.0.3",
+    "automerge-repo": "^0.0.50",
+    "automerge-repo-network-broadcastchannel": "^0.0.50",
+    "automerge-repo-network-websocket": "^0.0.50",
+    "automerge-repo-storage-localforage": "^0.0.50",
+    "automerge-repo-svelte-store": "./packages/automerge-repo-svelte-store"
```
Comment (ecstatic-morse) [inline packages/automerge-repo-demo-counter-svelte/package.json:19]: reminder: this should be a version number

Comment (acurrieclark) [reply packages/automerge-repo-demo-counter-svelte/package.json:19]: You should just be able to replace this with a version number and it works. If not, try also running `lerna link --force-local` from the repo root.

**Comment by acurrieclark** [comment]
Thanks for this, a really useful contribution.

I have flagged a few things for review. If you can get them patched up I will run CI and look at getting this merged.

**Comment by ecstatic-morse** [comment]
Added a README and did a `yarn upgrade @automerge/automerge` to pick up version 2.0.3 (which fixed #60).

File: packages/automerge-repo-svelte-store/src/index.ts [hunk 2]
```diff
@@ -0,0 +1,30 @@
+import type { Doc, ChangeFn } from '@automerge/automerge'
```
[the remark this answers is not in the document]
Comment (acurrieclark) [reply packages/automerge-repo-svelte-store/src/index.ts:1]: Thanks for the readme. If you could take a look at the formatting here I would be most grateful.

**Review by acurrieclark — CHANGES_REQUESTED** [review]

**Comment by acurrieclark** [comment]
Apologies, I forgot to press the “complete review” button earlier. Just the formatting to sort

**Comment by ecstatic-morse** [comment]
@acurrieclark should be fixed now. I updated the project level repo as well.

**Review by acurrieclark — APPROVED** [review]

**Comment by acurrieclark** [comment]
Thanks for this. Really appreciate the docs too.

**Comment by catdevnull** [comment]
hi! when is this being published into npm?

**Pull request merged**